### ISSUE TITLE
Content Management Framework - New OpenShift Project

### ISSUE THREAD

**Issue by ihallas** [BCDevOps/devops-requests#181]
## Step 0
**Are you the product owner or technical lead?**
If not, you will need to ask one of those two people to make this request.

**Have you had an alignment meeting?**
If not, please set up a meeting with the Enterprise DevOps branch leadership prior to submitting a project request.

## Step 1
Visit DevHub for detailed instruction on creating a request:
https://developer.gov.bc.ca/Getting-Started-on-the-DevOps-Platform/How-to-Request-a-New-OpenShift-Project

## Step 2
Make sure no duplicated request exists, search here:
https://github.com/BCDevOps/devops-requests/issues

## Step 3
Are you requesting to:
- [x] Create a project set
- [ ] Remove a project set

## Step 4
Fill out the following fields

* Short Team/Organization Name:  OSS -GDX
* Product Short Name: CMF
* Product Full Name:  Content Mangement Framework
* Product Description:  Re-Architect and move the existing CMF to the OpenShift platform
* Desired Environments: tools, dev, test, prod
* GitHub Repository (if exists): 

## Step 5
Submit this ticket and email us the contact information.

For security purpose, please provide the following contact information to the email address - pathfinder@gov.bc.ca

* link to the GitHub ticket you just created
* Product Technical Steward
  - GitHub ID: 
  - Business Email Address: 
  - RocketChat Account Username: 
* Product Owner
  - Business Email Address: (BCGov employee)
  - RocketChat Account Username: 
  - Organization Code:
  - Organization Unit Code: 
  - (refer to https://dir.gov.bc.ca/)

* BC Government Security Contact Information
  - Ministry Chief Information Officer:
  - (refer to https://www2.gov.bc.ca/assets/gov/british-columbians-our-governments/services-policies-for-government/information-technology/ocio/mcio_contact_list.pdf)

  - Ministry Information Security Officer:
  - (refer to https://intranet.gov.bc.ca/thehub/ocio/ocio-enterprise-services/information-security-branch/miso-contacts)

  - Ministry Privacy Officer:
  - (refer to https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/privacy/resources/privacy-officers)

Please note that we will only start processing your request when the email has been received.

**Comment by caggles** [comment]
Just waiting on a response to an email requesting more information!

**Comment by ihallas** [comment]
We have not received an invitation.  Should this come by email?
I asked in Rocketchat and it was suggested I ask in this ticket
THanks

**Comment by ShellyXueHan** [comment]
looks like your GitHub organization invitation was missing. I've just sent it out. After you have joined BCDevOps GitHub org, you will have access to the openshift console at https://console.pathfinder.gov.bc.ca:8443/ where you'll see the project set.

**Comment by ihallas** [comment]
As discussed in another ticket
This project has a typo in the Name
It is is currently Content Mangement Framework (*)

Instead of Content Management Framework
The ( a ) is missing in the word Management

**Comment by ShellyXueHan** [comment]
Please provide the namespace name.

**Comment by ihallas** [comment]
Content Mangement Framework (dev) - crmnaw-dev
Content Mangement Framework (test) - crmnaw-test
Content Mangement Framework (prod) - crmnaw-prod
Content Mangement Framework (tools) - crmnaw-tools

**Comment by ShellyXueHan** [comment]
Updated. pls double check.
